README.md

```diff
@@ -36,19 +49,30 @@ The config.inc.php settings format is:
 [ithenticate]
 
 ; Enable iThenticate to submit manuscripts after submit step 4
-;ithenticate = On
+; ithenticate = On
 
 ; Credentials can be set by context : specify journal path
-; The username to access the API (usually an email address)
-;username[MyJournal_path] = "user@email.com"
-; The password to access the API
-;password[MyJournal_path] = "password"
+; Unique turnitin api url
+; api_url[MyJournal_path] = ""
+; The API key to authorize the api calls
+; api_key[MyJournal_path] = ""
+
+; Unique ithenticate api url
+; Make sure to provice the schema(http:// or https://) with the url and no trailing slash
+; api_url = "https://some-ithenticate-account.com"
 
-; default credentials
-; The username to access the API (usually an email address)
-;username = "user@email.com"
+; The API key to authorize the api calls
+; api_key = "YOUR_API_KEY"
 
-; The password to access the API
-;password = "password"
+; Submission similarity check settings can also configured as global or journel level
+; To set a similarity check config on global level, just set the setting value as
+; addToIndex = On
+; To set a similarity check config on journale level, just set the setting value as
+; addToIndex[MyJournal_path] = On
+; See the README.md of plugin to see all possible similarity config settings
 ```
 
+## Restrictions
+1. Submission submitting user must confirm the iThenticate End User License Agreement to sent files to iThenticate service for plagiarims checking
+2. Upload of zip/tar/gzip is not allowed.
```

Comment:
"not allowed" might be confusing, as it suggests that the files will be rejected. Better to say instead that zip files are not sent to iThenticate. Are you sure this applies to `gzip`/`tar` as well?